dags/generate_dags.py

```diff
@@ -23,12 +25,25 @@ def generate_dags():
         key = file_["Key"]
         if key.endswith("/"):
             continue
+        file_name = Path(key).stem
         result = client.get_object(Bucket=bucket, Key=key)
-        collection = result["Body"].read().decode()
-        collection = json.loads(collection)
-        if collection.get("schedule"):
+        collections = result["Body"].read().decode()
```

Comment:
nit: could we call this something clunky like `discovery_configs` instead of collections which means something else the rest of the time?

Reply:
yessss